accname/name/comp_name_from_heading.tentative.html

```diff
@@ -0,0 +1,63 @@
+<!doctype html>
+<html>
+<head>
+  <title>Name Comp: Name From Content</title>
```

Comment:
per @MelSumner's comment, this title should have been name from heading, since this wpt is for the new naming mechanism.  Does this fix [help with your question](https://github.com/web-platform-tests/wpt/pull/44490#pullrequestreview-1882802430), Melanie?
```suggestion
  <title>Name Comp: Name From Heading</title>
```

Reply:
Commited, but leaving this comment thread unresolved as it's a reply to @MelSumner